docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
 
-**Save checkpoints folder after training**
+Also, note that you pass learning rate from the ``PARAMS`` dictionary.
 
 .. code-block:: python3
 
-    model_checkpoint = pl.callbacks.ModelCheckpoint(filepath='my/checkpoints')
+    class LitModel(pl.LightningModule):
+        def __init__(self):
+            super().__init__()
+            self.l1 = torch.nn.Linear(28 * 28, 10)
 
-    trainer = Trainer(logger=neptune_logger,
-                      checkpoint_callback=model_checkpoint)
-    trainer.fit(model)
+        def forward(self, x):
+            return torch.relu(self.l1(x.view(x.size(0), -1)))
 
-    neptune_logger.experiment.log_artifact('my/checkpoints')
+        def training_step(self, batch, batch_idx):
+            x, y = batch
+            y_hat = self(x)
+            loss = F.cross_entropy(y_hat, y)
+            return pl.TrainResult(loss)
 
-**Explicitly close the logger**
+        def configure_optimizers(self):
+            return torch.optim.Adam(self.parameters(), lr=PARAMS['LR'])
 
-Closing is optional but you may want to close it and then do something afterwards.
+Step 4 - Prepare Data Loader
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``DataLoader`` (you know it from PyTorch) is necessary to fit your model. Note that you pass ``batch_size`` from the ``PARAMS`` dictionary.
 
 .. code-block:: python3
 
-    neptune_logger.experiment.stop()
+    train_loader = DataLoader(MNIST(os.getcwd(), download=True, transform=transforms.ToTensor()),
+                              batch_size=PARAMS['batch_size'])
 
-Monitor your PyTorch Lightning training in Neptune
---------------------------------------------------
-Now you can watch your pytorch-lightning model training in Neptune!
+Step 5 - Create NeptuneLogger
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``NeptuneLogger`` is an object that integrates Neptune with PyTorch Lightning allowing you to track your experiments in Neptune. It's a part of the lightning library.
 
-Check out this |example experiment|.
-
-.. image:: ../_static/images/integrations/pytorch_lightning_monitoring.gif
-   :target: ../_static/images/integrations/pytorch_lightning_monitoring.gif
-   :alt: PyTorchLightning logging in neptune
-
-Full PyTorch Lightning monitor script
--------------------------------------
-Simply copy and paste it to ``pytorch_lightning_example.py`` and run.
-You can change your credentials in ``NeptuneLogger`` or run some tests as an anonymous user:
+In this minimalist example we use public user `"neptuner"`, who has public token: "ANONYMOUS".
 
 .. code-block:: python3
 
+    from pytorch_lightning.loggers.neptune import NeptuneLogger
+
     neptune_logger = NeptuneLogger(
         api_key="ANONYMOUS",
         project_name="shared/pytorch-lightning-integration",
-        ...
-        )
+        params=PARAMS)
+
+.. tip::
+
+    Make sure to use your API token in your projects. Read more about how to |token|.
+
+Step 6 - Fit model to the data - Neptune tracks it automatically
```

Comment:
The step should be pass logger to the trainer I think.

Reply:
I chanegd this step a bit.